ITEM.tsv

```diff
@@ -3129,13 +3129,13 @@ ITEM_20150317_003128	Torn Bat Wings
 ITEM_20150317_003129	Green Ointment
 ITEM_20150317_003130	A Vubbe ointment that is used to treat their injured. It's not possible to use it as is.
 ITEM_20150317_003131	Royal Tomb Guardian Power Source
-ITEM_20150317_003132	A power source used by the Guardian for movement.
+ITEM_20150317_003132	A power source used by the Guardian that allows it to move.
 ITEM_20150317_003133	Royal Tombs Magic Source
 ITEM_20150317_003134	The source of the magic that encircles the Royal Tombs.
 ITEM_20150317_003135	Broken Slate of the Royal Tomb
 ITEM_20150317_003136	A slate that ordered the Guardians to attack evil things. They are attacking everyone because it is broken.
 ITEM_20150317_003137	Burning Stone
-ITEM_20150317_003138	The stone that light stone lanterns. Although it is burning, it isn't very hot.
+ITEM_20150317_003138	A stone that lights stone lanterns. Although it is burning, it isn't very hot to the touch.
```

Comment:
Unnecessary adding of "to the touch." Try to not over-complicate text when it doesn't need to be.